src/config.js

```diff
@@ -1,23 +1,23 @@
+import deepmerge from "deepmerge";
```

Comment:
do we really need an extra dependency for that? What about using a recursive util function?

Reply:
You can write one yourself if you want, but someone else did the work for us, so why not use it ^^

Reply:
Yeah, it's up to you. I personally don't like to have hundreds of dependencies for simple things. But we can go with this approach. What do you think @noctera ?

Reply:
I think in this case it should be OK, but if you want to rewrite it yourself on the long term, it is up to you. I think the dependency is not that big.

I'm a fan of less dependencies too, but if you can safe time and afford you have to think about the cost-benefit factor.